skglm/solvers/cd_solver.py

```diff
@@ -214,11 +239,11 @@ def cd_solver(
 
     Returns
     -------
-    alphas : array, shape (n_alphas,)
-        The alphas along the path where models are computed.
+    w : array, shape (n_features,)
+        Coefficients.
 
-    coefs : array, shape (n_features, n_alphas)
-        Coefficients along the path.
+    obj_out : array, shape (n_iter,)
```

Comment:
do we really return this? or the optimality condition violation instead

Reply:
We do return this. See L371.